frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -148,6 +132,23 @@ export function usePrimaryFilterDimensions(): UseQueryResult<PrimaryFilterDimens
   );
 }
 
+export const TEMP_ALLOW_NAME_LIST = ["Homo sapiens", "Mus musculus"];
```

Comment:
one last nit then LGTM! I think we can get rid of this export statement, since no one else is using this constant?